lua/doom/modules/features/symbols/init.lua

```diff
@@ -19,19 +19,17 @@ symbols.settings = {
 symbols.packages = {
   ["symbols-outline.nvim"] = {
     "simrat39/symbols-outline.nvim",
-    commit = "6a3ed24c5631da7a5d418bced57c16b32af7747c",
     cmd = {
       "SymbolsOutline",
       "SymbolsOutlineOpen",
       "SymbolsOutlineClose",
     },
-    opt = true,
   },
 }
 
 symbols.configs = {}
 symbols.configs["symbols_outline.nvim"] = function()
-  vim.g.symbols_outline = doom.features.symbols.settings
+  require("symbols-outline").setup(doom.features.symbols.settings)
```

Comment:
It looks like the configuration function is never called, @NTBBloodbath do you have an idea why?

When running `:lua require("symbols-outline").setup()` symbols outline opens correctly

Reply:
Nevermind, the configuration function was never called because the module name was not correct. Ie: `-` vs. `_`.